README.md

```
@@ -87,4 +87,19 @@ The goal is to ensure that the threads coordinate properly using synchronization
 - The output strictly follows the order: **A → B → C → A → B → C...**
 
 #### [Printer.java](src/main/java/com/anusruta/multithreading/printabc/Printer.java)
-#### [PrintABCTest.java](src/test/java/com/anusruta/multithreading/printabc/PrintABCTest.java)
\ No newline at end of file
+#### [PrintABCTest.java](src/test/java/com/anusruta/multithreading/printabc/PrintABCTest.java)
+
+## Question - 4
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)